library/blivet.py

```diff
@@ -1265,18 +1289,114 @@ def _manage_encryption(self, members):
 
         return managed_members
 
+    def _manage_thin_pools(self, pool_device):
+
+        def get_vol_size(volume):
```

Comment:
The parameter name should be `vol_size`, or `vol_size_str` if you think that would be more descriptive.